profiles/killbill/src/test/resources/killbill.properties

```diff
@@ -21,6 +21,8 @@ org.killbill.overdue.uri=overdue.xml
 
 org.killbill.payment.retry.days=8,8,8
 
+org.killbill.payment.plugin.timeout=8s
```

Comment:
I think it would be better to see if we could overwrite the timeout value for that test specifically to something even smaller:

* `8 sec` is still quite long for a test that runs every time
* Also, that impacts all out tests (where default timeout is now 8 sec)